target/linux/ramips/image/mt7620.mk

```
@@ -207,6 +207,16 @@ define Device/comfast_cf-wr800n
 endef
 TARGET_DEVICES += comfast_cf-wr800n
 
+define Device/dlink_dap-1620-a2
+  DEVICE_VENDOR := D-Link
+  DEVICE_MODEL := DAP-1620
+  IMAGE_SIZE := 7872k
+  DEVICE_VARIANT := A2
+  SOC := mt7620a
+  DEVICE_PACKAGES := kmod-mt76x0e kmod-mt76x2 rssileds
```

This board only have MT7612.
```suggestion
  DEVICE_PACKAGES := kmod-mt76x2 rssileds
```

---

Had some issues without this driver. Also most devices in the 7620 series used this driver. So was thinking its default/required. After removing it, didn't notice any difference in functionality. Changes made and committed. thank you!